drivers/platform/stm32/stm32_spi.c

```diff
@@ -633,7 +633,12 @@ void stm32_spi_dma_callback(struct no_os_dma_xfer_desc *old_xfer,
 	sdesc->stm32_spi_dma_done = true;
 
 	/* Dummy read to clear any pending read on SPI */
+#ifndef SPI_SR_RXNE
```

Comment:
i see that the newer chips don't have this bit in the SR register so this is your method of distinguishing SPI peripherals for newer chips like H5 from the older chips like F4

could you add a comment on this line explaining this ?

Reply:
The file already utilizes the SPI_SR_RXNE and SPI_SR_TXE bits to distinguish between the chips. Given this consistent usage, is the comment needed?

Reply:
i just saw what Ramona wrote here:
https://github.com/analogdevicesinc/no-OS/blame/57499efc27ba9980f3856ceee1dd629062a1c4d5/drivers/platform/stm32/stm32_spi.c#L387-L390

no need anymore :D